### PR TITLE
Note default composer options in deployment file

### PR DESCRIPTION
**Pull request by weshooper**
Need verifying Fortrabbit side, based on guesswork and assumptions! :-)

(may be nice, for completeness, to note that `no-suggest` appears to be on by default too)

### CODE REVIEW

**Comment by ostark** [comment]
Thanks Wes,
the "double negation" can be very confusing - thanks for your contribution!

**Comment by weshooper** [comment]
@ostark it can, heh! I've added some additional knowledge on the pre script - I'd learnt it before ages ago, but just needed to relearn/remember it...

File: docs/deployment-file-v2.md [hunk 1]
```diff
@@ -38,16 +38,16 @@ composer:
...
-    # Resolves to the --no-plugins parameter
+    # Resolves to the --no-plugins parameter, default is false
     no-plugins: false
 
-    # Resolves to the --no-scripts parameter
+    # Resolves to the --no-scripts parameter, default is false
     no-scripts: false
 
-# called after Composer runs
+# called after Composer runs (must be a single PHP script)
```
Comment (ostark) [inline docs/deployment-file-v2.md:50]: these additions make it crystal clear!

File: docs/deployment-file-v2.md [hunk 2]
```diff
@@ -71,8 +71,32 @@ post: post.php
...
 
 Assuming you are using two Apps, `your-app-prod` and `your-app-stage`, you can setup two deployment files with named: `fortrabbit.your-app-prod.yml` and `fortrabbit.your-app-stage.yml`. This way, you can have both deployment files in a single repo but which one is to be used is determined by the App you deploy to.
+
+## Private composer use case
+
+The need for a pre script is rare, but if you [use private repositories](/private-composer-repos) over SSH (i.e. rather than `github-oauth`) and are concerned about a (highly unlikely) MITM attack, you can add a simple pre script as follows:
+
+```php
+<?php
+// Prevent MITM, by enabling StrictHostKeyChecking and adding a verified host for github.com before composer SSH installs
```
Comment (ostark) [inline docs/deployment-file-v2.md:90]: edge case?

Comment (weshooper) [reply docs/deployment-file-v2.md:90]: 100% agree! We got the [original code in a gist](https://gist.github.com/ukautz/3b798fde25e38a4d05476d0c2675a105) through a support chat so figured was something that has been asked before and may be helpful, but you guys will know better than I whether it's useful enough to include here...

This originated from https://help.fortrabbit.com/private-composer-repos recommending SSH keys for private repositories, would you suggest `github-oauth` in `composer.json` instead, which I assume would eliminate this edge case?

**Comment by frank-laemmer** [comment]
@ostark can this be closed?

**Comment by weshooper** [comment]
@frank-laemmer I had forgotten about this! Would removing the private composer edge case section help? Appreciate that may not be as useful as the rest 😊

**Pull request merged**